### PR TITLE
Feature/authentication

### PR DESCRIPTION
**Pull request by zanetasochon**

### CODE REVIEW

File: src/App.tsx
Comment (paulboguta) [inline src/App.tsx:1]: suggestion: you can create routes object
https://reactrouter.com/en/main/hooks/use-routes

File: src/services/auth.service.ts
Comment (paulboguta) [inline src/services/auth.service.ts:1]: fix: Auth interface (IAuth) is in 3 files: auth.service.ts, SignIn.tsx, SignUp.tsx move it to new folder (types) and new file (suggestion: user.types.ts) and then import.

File: src/store/store.ts
Comment (paulboguta) [inline src/store/store.ts:1]: You can delete those comments

**Review by paulboguta — CHANGES_REQUESTED** [review]
great job, 3 things to fix

File: src/App.tsx
Comment (zanetasochon) [reply src/App.tsx:1]: Done 💪

File: src/store/store.ts
Comment (zanetasochon) [reply src/store/store.ts:1]: Done 💪

File: src/services/auth.service.ts
Comment (zanetasochon) [reply src/services/auth.service.ts:1]: Done 💪

**Pull request merged**